### PR TITLE
isort clone of flake8 wrapper to run on individual files.

### PR DESCRIPTION
**Pull request by machallboyd**
A package to run isort on individual files, which should respect the appropriate configs.

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: isort_per_file/__init__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import os.path
+import pathlib
+import subprocess
+import sys
+
+
+
+def main():
+    my_flake8 = os.path.join(os.path.dirname(sys.argv[0]), 'flake8')
```
Comment (npilon) [inline isort_per_file/__init__.py]: You want this to be `my_isort`

**Review by npilon — CHANGES_REQUESTED** [review]
I think this is executing the wrong binary. You'll also need to add a pre-commit hook: https://github.com/LexMachinaInc/precommit-per-file-flake8/blob/main/.pre-commit-hooks.yaml

File: isort_per_file/__init__.py (hunk 1 above)
Comment (machallboyd) [reply isort_per_file/__init__.py]: Can I test this before merging?

Comment (npilon) [reply isort_per_file/__init__.py]: Yes! I think you can edit your `.pre-commit-config.yaml` in `deus_lex` to invoke this branch, though you do have to then do a bit of extra work to tell it that a new version is available. You might also be able to: https://pre-commit.com/#pre-commit-try-repo

**Review by npilon — APPROVED** [review]
@machallboyd I've made what I think are the correct adjustments to `pyproject.toml` for this and regenerated the lock-file - can you take a quick look and confirm?

**Comment by machallboyd** [comment]
> 

Looks ok. Is there a reason we're bumping the point version of python?

File: pyproject.toml [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 name = "flake8_per_file"
-version = "0.5.0"
+version = "0.5.1"
 description = "Per file flake8"
 authors = ["Gavin Carothers <gavin@carothers.name>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.1"
```
Comment (machallboyd) [inline pyproject.toml:9]: What's the bump here for?

Comment (npilon) [reply pyproject.toml:9]: `isort` advertises itself as requiring python 3.6.1 and newer for some reason.

Comment (machallboyd) [reply pyproject.toml:9]: Weird. No comments otherwise.

**Pull request merged**